index.html

```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
+                        </div>
+                        <ul class="skills">
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javasript</li>
+                        </ul>
+                        <div class="read-more">
+                            <a href="#">
+                                See Project
+                            </a>
+                        </div>
+                    </div>
+                </article>
+            </section>
+            <section id="about-me-more" class="about-me-more">
                 <div>
                     <div>
-                        <h1>
-                            I'm William
-                            Glad to see you!
-                        </h1>
+                        <h2>
+                            About <br />
+                            Myself
+                        </h2>
                     </div>
                     <div class="description">
-                        <p>I'm a software developer! I can help you build a product , feature or website Look through
-                            some
-                            of my
-                            work and
-                            experience! If you like what you see and have a project you need coded, don’t hestiate to
-                            contact
-                            me.</p>
+                        <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+                        experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
                     </div>
                     <div class="connect">
                         <h5>LET'S CONNECT</h5>
                         <ul>
-                            <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
-                            <li><a href="#"><img src="assets/img/linkedin.png" alt="linkedin" /></a></li>
-                            <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
                             <li><a href="#"><img src="assets/img/github.png" alt="github" /></a></li>
+                            <li><a href="#"><img src="assets/img/linkedin.png" alt="linkedin" /></a></li>
                             <li><a href="#"><img src="assets/img/angies.png" alt="angies" /></a></li>
+                            <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
+                            <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
                         </ul>
                     </div>
+                    <div class="read-more">
+                        <a href="#">
+                            Get my resume
+                        </a>
```

Comment:
I can see you haven't implemented the given states for your buttons, kindly see the images below and implement the given states for all your buttons.
<img width="348" alt="btnstate" src="https://user-images.githubusercontent.com/64643995/212485956-125bff80-8ce5-4883-8f9d-354ab0549183.png">